CHANGES.md

```diff
@@ -13,6 +13,7 @@ Unreleased changes refer to our current [master branch](https://github.com/openc
 * Fixed `Too many connections to ldap` when openchange runs on samba as member of a domain.
 * Fixed `Mark All as Read` feature regression bug introduced by 7737bdf6
 * Address book working much better than before
+* ResolveNames and ResoveNamesW operations perform a better Ambiguous Name Resolution search, which returns more matches.
```

Comment:
@javierrag: suggestion for the entry. It is fair to call it an improvement rather than a fix or performances? If so, I would suggest to add a new Improvements section in CHANGES.md

With regards to the text, what do you think of the following suggestion: 

### Improvements
* More records returned when searching for ambiguous names.